ssl-config/src/test/resources/readme.md

```diff
@@ -0,0 +1,65 @@
+# Test Keys and Certificates
+
+This directory contains certificates and keys in various formats that are used to test the package.
+
+## Test CA
+
+The certificate authority for this set of test certificates.
+
+### Raw files
+
+File | Type | Format | Password | Signed By | Common Name
+---- | ---- | ------ | -------- | --------- | -----------
+testCA/testCA.key | Private key | PEM | testtest | | 
```

Comment:
(I use Chrome's "Markdown Preview Plus" which makes it very easy to assess whether it's correct or not without making a PR.)